perf_dashboard/history/templates/history.html

```diff
@@ -62,7 +62,9 @@ <h2>History of Istio Performance and Scalability</h2>
             <div class="card-body">
               <p>Snapshots of performance and scalability page of the istio.io website for each release.</p>
               <ul>
-                  <li><a href="https://istio.io/docs/ops/deployment/performance-and-scalability/" target="_blank">Current Release</a></li>
+                  <li><a href="https://istio.io/latest/docs/ops/deployment/performance-and-scalability/" target="_blank">Current Release</a></li>
+                  <li><a href="https://istio.io/v1.7/docs/ops/deployment/performance-and-scalability/" target="_blank">1.7</a></li>
+                  <li><a href="https://istio.io/v1.6/docs/ops/deployment/performance-and-scalability/" target="_blank">1.6</a></li>
                   <li><a href="https://archive.istio.io/v1.5/docs/ops/deployment/performance-and-scalability/" target="_blank">v1.5</a></li>
```

Comment:
For consistency, and I'n not sure when/if the archive site will go away, the older <=1.5 should be changed to use istio.io/v1.x.